src/lib/components/select/select.md

```diff
@@ -51,3 +51,85 @@ const selectOptions = [
   isMulti
 />;
 ```
+
+```js
+initialState = {
+  groupValue: [],
+};
+
+function updateValue(name, value) {
+  setState({ [name]: value });
+}
+
+const selectOptions = [
+  {
+    label: 'Group 1!',
+    options: [
+      { value: 'Option 1', label: 'Option 1', group: 'Group 1' },
+      { value: 'Option 2', label: 'Option 2', group: 'Group 1' },
+      { value: 'Option 3', label: 'Option 3', group: 'Group 1' },
+    ],
+    value: '',
+  },
+  {
+    label: 'Group 2!',
+    options: [
+      { value: 'Option 4', label: 'Option 4', group: 'Group 2' },
+      { value: 'Option 5', label: 'Option 5', group: 'Group 2' },
+      { value: 'Option 6', label: 'Option 6', group: 'Group 2' },
+    ],
+    value: '',
+  },
+  {
+    label: 'Group 3!',
+    options: [
+      { value: 'Option 7', label: 'Option 7', group: 'Group 3' },
+      { value: 'Option 8', label: 'Option 8', group: 'Group 3' },
+      { value: 'Option 9', label: 'Option 9', group: 'Group 3' },
+    ],
+    value: '',
+  },
+];
+
+<Select
+  isMulti
+  withGroups
+  placeholder="Country?"
+  classNames="select-auto-complete-country"
+  className="select-auto-complete-country"
```

Comment:
are these styles important? they kinda clutter the example

Reply:
Done